.yarn/versions/86a2ae3f.yml

```
@@ -0,0 +1,2 @@
+releases:
+  "@yarnpkg/eslint-config": minor
```

I think the change of resolution behaviour makes it a major
```suggestion
releases:
  "@yarnpkg/eslint-config": major
```

---

It's a breaking change yes, but marking it as a major shows the next version as `1.0.0` when it should be `0.3.0`